CDCgov/NEDSS-Modernization
# [CNFT1-1227] Address 2 and use to Address for patient

## Description

Adds the `use` and `address2` fields to the Patient Search and Patient Summary API.

## Tickets

* [CNFT1-1227](https://cdc-nbs.atlassian.net/browse/CNFT1-1227)
* [CNFT1-1225](https://cdc-nbs.atlassian.net/browse/CNFT1-1225)

## Steps to verify

1. Search for a Patient
2. Click the name of the Patient to enter the Patient Profile
3. Navigate to the Demographics tab
4. Add a new House / Home Address with an address 2 field
5. Return to search and search for the Patient
6. Ensure the new Address is in the search result and contains the address 2 field

In the search results
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/70d55b10-69ac-4277-80c8-74c22ea7f8d3)

On the Patient Profile Summary Card
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/1ccc5a99-57c6-4e21-81c2-570581890b39)


[CNFT1-1227]: https://cdc-nbs.atlassian.net/browse/CNFT1-1227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1225]: https://cdc-nbs.atlassian.net/browse/CNFT1-1225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
